frontend/src/utils/toast.utils.tsx

```diff
@@ -79,6 +79,58 @@ export const TOASTS = {
       ERROR: 'REDPANDA_CONNECT_LINT_CONFIG_ERROR',
     },
   },
+  TRANSFORM: {
```

Comment:
I am not sure this is the best way to separate this, don't see the value in the cloud-ui. I have seen some degration like closing one toast and seeing all being closed.

Reply:
Fixed.